Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -588,11 +643,7 @@ void CGContextAddLines(CGContextRef context, const CGPoint* points, unsigned cou
         return;
```

Comment:
Either is fine.

---
In reply to: [85367100](https://github.com/Microsoft/WinObjC/pull/1240#discussion_r85367100) [](ancestors = 85367100)